recipes/rpfits/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "rpfits" %}
+{% set version = "2.23" %}
+{% set sha256 = "7fbed9951b16146ee8d02b09f447adb1706e812c33a1026e004b7feb63f221a0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://newton.cx/~peter/files/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  skip: true  # [win]
+
+requirements:
+  build:
+    - gcc  # [osx]
+    - libgfortran  # [unix]
+    - toolchain
+  run:
+    - libgfortran  # [unix]
+
+test:
+  commands:
+    - rpfex </dev/null
+    - test -f $PREFIX/lib/librpfits.so  # [linux]
+    - test -f $PREFIX/lib/librpfits.dylib  # [osx]
```

Comment:
Now that we are using `conda-build 2` we can use:

```yaml
    - conda inspect linkages -p $PREFIX rpfits  # [not win]
    - conda inspect objects -p $PREFIX rpfits  # [osx]
```

I strongly encourage you to add that to package like this one. It helps a lot to debug linking issues.